src/components/annotations/annotation-utilities.ts

```diff
@@ -11,6 +11,10 @@ export function normalizeAngle(angle: number) {
 }
 
 // Returns the default peak for a sparrow from sourceX, sourceY to targetX, targetY.
+const minPeakPercent = 1;
```

Comment:
Minor, but some renaming or comments could make this more clear.
* minPeakPercent, maxPeakPercent --> not really percentages, and min is confusingly larger than max. Maybe rename to something like short/longSparrowPeakHeight ?
* minLength, maxLength --> short/longSparrowLength ?
* radius --> is just length, I think?